pom.xml

```diff
@@ -27,7 +27,7 @@
   -->
   <artifactId>google-storage-plugin</artifactId>
   <packaging>hpi</packaging>
-  <version>1.1-SNAPSHOT</version>
+  <version>1.1</version>
```

Comment:
This is being updated as part of the release process, please do not update manually

Reply:
Why it's would cause any problems ? It's can be updated.